### PR TITLE
Added to grafana plugin docs how to connect to k8s service

### PR DESCRIPTION
**Pull request by jcr179**

### CODE REVIEW

File: docs/grafana-flamegraph-plugin.md [hunk 1]
```diff
@@ -22,6 +22,8 @@
 4. Select the `Parca` datasource.
 5. Enter the API URL of the Parca server in the `API Endpoint` field. For example, `http://localhost:7070/api`.
    Note: Please make sure cors configuration of the Parca server allow requests from your Grafana Dashboard origin. If you Grafana dashboard is running at `http://localhost:3000`, then ensure that the Parca server is started with either `--cors-allowed-origins='http://localhost:3000'` or `--cors-allowed-origins='\*'` flag. Please refer the [docs](https://www.parca.dev/docs/grafana-datasource-plugin#allow-cors-requests).
+   
+   If you have [deployed Parca in Kubernetes](https://www.parca.dev/docs/kubernetes), you can enter the ClusterIP of the Parca service created when applying the Parca server manifest to your cluster in the `API Endpoint` field. The ClusterIP can be found using `kubectl get service -n parca parca`.
```
Comment (maxbrunet) [inline docs/grafana-flamegraph-plugin.md:26]: You should not need to know the ClusterIP, DNS is here for that :wink:: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#services

TL;DR use `http://parca.parca.svc.cluster.local/api`

Comment (jcr179) [reply docs/grafana-flamegraph-plugin.md]: you're absolutely right. Thank you :) New commit reflects this better method.
